README.md

```diff
@@ -23,6 +23,26 @@ zongji.start({
 });
 ```
 
+## Subscribe Binary Log Position
+
+```javascript
+var zongji = new ZongJi({ /* ... MySQL Connection Settings ... */ });
+
+// Receive `binlog_position` events
+zongji.on('binlog_position', function(pos) {
+  console.log(pos);
+  // { filename: 'mysql-bin.000001', position: 77656182 }
+});
+
+// Need both `subscribePosition` and `rotate` to take effect
+zongji.start({
+  subscribePosition: true,
+  includeEvents: ['tablemap', 'writerows', 'updaterows', 'deleterows', 'rotate']
+});
+```
+> Notice: `pos.filename` won't be updated when `rotate` event is disabled
```

Comment:
We should always receive rotate event when we subscribed position event. Otherwise, it sounds like a bug.